server/Arcadia.Assistant.Web/Infrastructure/AkkaTimeoutExceptionHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Arcadia.Assistant.Web.Infrastructure
+{
+    using System;
+    using System.Threading.Tasks;
+
+    using Akka.Actor;
+
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Http.Extensions;
+
+    using NLog;
+
+    public class AkkaTimeoutExceptionHandler : IMiddleware
```

Comment:
```suggestion
    public class AkkaTimeoutExceptionHandlerMiddleware : IMiddleware
```
?

Reply:
No. It's middleware that handles errors, hence ExceptionHandler. You don't call List a ListCollection